### ISSUE TITLE
Document that --csr doesn't save a lineage and resulting cert can't be renewed

### ISSUE THREAD

**Issue by schoen** [certbot/certbot#3690]
https://community.letsencrypt.org/t/how-to-renew-certificates-generated-from-a-csr/21684/9

The behavior is described at https://github.com/certbot/certbot/blob/master/certbot/main.py#L526

In https://certbot.eff.org/docs/using.html, I don't see any reference to this except for a copy of the output from `--help`, which doesn't explain this.

**Comment by bmw** [comment]
This shouldn't be too hard to do. We could just update the `--help` text and add some documentation to `docs/using.rst`.

**Comment by bmw** [comment]
I think we should still do this.

**Comment by ASChism** [comment]
I recently found you can generate a cert using the secp384r1 curve and then get it signed using certbot --csr. Unfortunately, I am running into this same issue now. The cert I obtained this way cannot be renewed by certbot.

**Comment by komailrk** [comment]
I was using `--csr` and did not know it would not be able to renew the certificate. Additionally, when using `--csr` it places the certificates in the `cwd`.

I am fairly new to `certbot` and was hoping I found a reference somewhere warning me about it. 

For those running into this and still looking to automate certificate deployment using CSR, checkout acme.sh. Keep in mind my use case was to deploy certificates for internal hosts and that's the only time I ran into the problem where certbot didn't fit my needs.
